char.c

```diff
@@ -27,8 +27,16 @@
 #include "file.h"
 #include "char.h"
 
+#if defined(__FreeBSD__)
+#define _DONT_USE_CTYPE_INLINE_
+#include <runetype.h>
+#endif 
 #include <ctype.h>
 
+#if defined(__FreeBSD__)
```

Comment:
No need for "defined()" in single-condition if. Just use ifdef like you do in other files.